modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -6938,7 +6938,10 @@ public VirtualHostModel newVirtualHostModel() {
 				BenchmarksPropsValues.ADMIN_INSTANCE_VIRTUAL_HOST_NAME);
 		}
 		else {
-			virtualHostModel.setHostname(_webId);
+			virtualHostModel.setHostname(
```

Comment:
@slnn Doing this will cause the company to have a webId different with its virtual host name, if the property is set to something other than `liferay`, because `liferay` is hard-coded in `newCompanyModels()`.

Reply:
Yes, portal allows this: use different values for webid of company and virtualhost of VirtualHost.